changelog/fix_update_layoutspacebeforecomment_to.md

```diff
@@ -0,0 +1 @@
+* [#9984](https://github.com/rubocop/rubocop/pull/9984): Fix false negatives involving heredocs for `Layout/SpaceBeforeComma`, `Layout/SpaceBeforeComment`, `Layout/SpaceBeforeSemicolon` and `Layout/SpaceInsideParens`. ([@dvandersluis][])
```

Comment:
It seems that `Layout/SpaceBeforeComma` is duplicated :-)

```suggestion
* [#9984](https://github.com/rubocop/rubocop/pull/9984): Fix false negatives involving heredocs for `Layout/SpaceBeforeComma`, `Layout/SpaceBeforeSemicolon`, and `Layout/SpaceInsideParens`. ([@dvandersluis][])
```

Reply:
One is `Comma` and one is `Comment`, they’re different cops 😁

Reply:
Oops! I'm sorry about I made a terrible mistake...

Reply:
No worries thanks for checking!